media/stylus/includes/vars.styl

```diff
@@ -173,7 +173,7 @@ $selector-heading-font-fallback = 'html[data-ffo-opensanslight="false"]:not(.no-
 paths
 ====================================================================== */
 $path-to-assets = '/media/';
-$path-to-images = $path-to-assets + 'img/';
+$path-to-images = '/static/img/';
```

Comment:
That's great! I was wondering about how to get staticfiles to replace paths. Thanks for the info.